lib/vue/src/components/input/CInputCheckbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div
+      class="d-flex"
+      style="gap: 1.5rem;"
+    >
+      <h6
+        class="mb-0 text-primary"
+        :class="{ 'text-muted': value }"
+      >
+        No
+      </h6>
+      <b-form-checkbox
+        v-bind="$attrs"
+        v-model="value"
+        switch
+        @input="handleInput"
```

Comment:
@input will trigger on non user interaction aswell, so I'm not sure you want this event